### PR TITLE
Move display package to nextjs fork

### PR DESCRIPTION
**Pull request by beerose**
Part of https://github.com/blitz-js/blitz/issues/2945

I haven't modified or deleted display package functions yet.

## Bug Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)

## Feature Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)
- [ ] Documentation added/updated (submit PR to [blitzjs.com repo `canary` branch](https://github.com/blitz-js/blitzjs.com/tree/canary))

### CODE REVIEW

**Review by imcodingideas — APPROVED** [review]
lgtm

File: nextjs/packages/next/package.json [hunk 1]
```diff
@@ -111,6 +112,7 @@
     "node-libs-browser": "^2.2.1",
     "npm-which": "^3.0.1",
     "null-loader": "4.0.1",
+    "ora": "^5.3.0",
```
Comment (beerose) [inline nextjs/packages/next/package.json:115]: Probably can import from next/dist/compiled/ora

File: nextjs/packages/next/server/lib/logging.ts [hunk 2]
```diff
@@ -65,3 +69,148 @@ export const baseLogger = (options?: ISettingsParam): Logger => {
...
+// Using bright brand color so it's better for dark terminals
+const brandColor = blitzBrightBrandColor
+
+const withBrand = (str: string) => {
+  return c.hex(brandColor).bold(str)
+}
+
+const withWarning = (str: string) => {
+  return `⚠️  ${c.yellow(str)}`
+}
```
Comment (flybayer) [inline nextjs/packages/next/server/lib/logging.ts:88]: remove in favor of tslog.warn

File: nextjs/packages/next/server/lib/logging.ts [hunk 3]
```diff
@@ -65,3 +69,148 @@ export const baseLogger = (options?: ISettingsParam): Logger => {
...
+  return `${c.gray('>')} ${str}`
+}
+
+const withCheck = (str: string) => {
+  return `${c.green('✔')} ${str}`
+}
+
+const withX = (str: string) => {
+  return `${c.red.bold('✕')} ${str}`
+}
```
Comment (flybayer) [inline nextjs/packages/next/server/lib/logging.ts:100]: remove in favor of tslog.error

File: nextjs/packages/next/server/lib/logging.ts [hunk 4]
```diff
@@ -65,3 +69,148 @@ export const baseLogger = (options?: ISettingsParam): Logger => {
...
+
+/**
+ * Logs a red error message to stderr.
+ *
+ * @param {string} msg
+ */
+const warning = (msg: string) => {
+  console.log(withCaret(withWarning(msg)))
+}
```
Comment (flybayer) [inline nextjs/packages/next/server/lib/logging.ts:141]: remove in favor of tslog.warn

**Pull request merged**